src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -217,6 +222,8 @@ private void visitReturn(DetailAST ast) {
         private int count;
         /** Maximum allowed number of return statements. */
         private Integer maxAllowed;
+        /** Identifies if context (method/lambda/constructor) returns anything. */
```

Comment:
This javadoc makes me feel like if `isVoidContext` is `true` then context returns something, but it should have an oppposite meaning. Please a change to `Identifies if context is void.` - I think that is enough clear enough.

Reply:
corrected